packages/core-ethereum/crates/core-ethereum-db/src/db.rs

```diff
@@ -162,6 +182,7 @@ impl<T: AsyncKVStorage<Key = Box<[u8]>, Value = Box<[u8]>> + Clone> HoprCoreEthe
             count += 1;
         }
 
+        batch_ops.del(unrealized_balance_key);
```

Comment:
Why should the unrealized balance be reset if we remove invalid tickets? This operation should not concern the unrealized balance at all.

Reply:
So should this then be dropped?